subsys/caf/modules/Kconfig.ble_bond

```diff
@@ -11,6 +11,70 @@ menuconfig CAF_BLE_BOND
 	depends on CAF_SETTINGS_LOADER
 	depends on CAF_BLE_COMMON_EVENTS
 	select CAF_BLE_BOND_SUPPORTED
+	help
+	  The simple CAF implementation that supports only bond erase for default Bluetooth local
+	  identity when predefined button click is detected (if enabled with Kconfig).
+
+if CAF_BLE_BOND
+
+config CAF_BLE_BOND_PEER_ERASE_CLICK
+	bool "Erase bonds using selected button click"
+	depends on CAF_CLICK_EVENTS
+	help
+	  To remove bonds, predefined click type of a predefined button must be detected.
+
+config CAF_BLE_BOND_PEER_ERASE_CLICK_KEY_ID
+	hex "Key ID of a button click used to erase bonds"
+	range 0x0000 0xffff
+	depends on CAF_BLE_BOND_PEER_ERASE_CLICK
+
+config CAF_BLE_BOND_PEER_ERASE_CLICK_TIMEOUT
+	int "Click detection after system start timeout [ms]"
+	default -1
+	depends on CAF_BLE_BOND_PEER_ERASE_CLICK
+	help
+	  This Kconfig option limits time of waiting for the button click detection after boot.
+	  Negative value disables the time limit. A non-negative value sets time limit to defined
+	  value.
+
+	  The timeout may need to be longer for some applications (for example if click event
+	  submission or processing is delayed by operations in system workqueue context).
+
+choice
+	prompt "Select click type"
+	depends on CAF_BLE_BOND_PEER_ERASE_CLICK
+	default CAF_BLE_BOND_PEER_ERASE_CLICK_DOUBLE
+	help
+	  The click types are directly mapped from the click enum in click_event.
+
+config CAF_BLE_BOND_PEER_ERASE_CLICK_NONE
+	bool "None click"
+	help
+	  Button pressed and held for a period of time that is too long for short click, but too
+	  short for long click.
```

Comment:
Do we need to have this? This "none" click is actually "not short, nor long" click type.

Reply:
I would remove this, its weird for me.

Reply:
Indeed it looks wrong. From the title I expected it to be something like a button press without any click event but help makes it weird. Let's remove it.

Reply:
Issue was introduced by the naming coincidence. I removed the none click.